services/export_test.go

```diff
@@ -11,7 +11,8 @@ func ExportedExecuteRunAtBlock(
 	overrides models.RunResult,
 	blockNumber *models.IndexableBlockNumber,
 ) (models.JobRun, error) {
-	return executeRunAtBlock(run, store, overrides, blockNumber)
+	run.Overrides = overrides
```

Comment:
I'm hesitant to do anything in these functions besides delegate to the private method. These are tests helpers to expose private functions in tests. Can you expand on why this is required?

Reply:
I may have left this accidentally, I don't believe it should be required.